Yank/commands/prepare.py

```diff
@@ -71,7 +71,7 @@
   --platform=PLATFORM           OpenMM Platform to use (Reference, CPU, OpenCL, CUDA)
   --precision=PRECISION         OpenMM Platform precision model to use (for CUDA or OpenCL only, one of {mixed, double, single})
   --pressure=PRESSURE           Pressure for simulation (in atm, or simtk.unit readable string) [default: 1*atmospheres]
-  --restraints=TYPE             Restraint type to add between protein and ligand in implicit solvent (harmonic, flat-bottom) [default: flat-bottom]
+  --restraints=TYPE             Restraint type to add between protein and ligand in implicit solvent (Harmonic, FlatBottom)
```

Comment:
Didn't we deprecate the `prepare` command? Or is that happening in a different PR?